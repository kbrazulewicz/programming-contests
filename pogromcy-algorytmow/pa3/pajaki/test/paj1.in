6
3
1 2 2 3 3 1
3
2 3 1 2 3 1
4
4 2 4 3 1 4 3 1 2 3
4
1 2 4 3 2 4 4 1 3 1
14
1 11 12 1 2 14 2 3 2 13 2 8 3 6 3 14 3 5 13 3 4 13 4 7 8 4 9 4 5 12 5 11 6 5 5 14 6 11 7 8 7 9 7 10 8 13 9 10 12 11
14
1 13 1 14 12 1 2 7 8 2 3 13 14 3 4 3 4 8 4 14 9 4 5 12 10 5 5 11 5 6 6 12 6 13 11 6 7 9 7 8 8 9 14 9 10 11 12 13 14 13
4
4 2 4 3 1 4 3 1 2 3
4
1 2 4 3 2 4 4 1 3 1
18
17 18 1 17 18 1 9 17 18 16 17 16 2 8 2 9 2 10 3 10 7 3 14 3 3 11 6 4 4 11 4 13 4 12 5 13 12 5 6 14 13 6 11 6 7 15 8 7 7 10 7 14 8 16 8 9 8 10 15 8 16 9 14 11 13 12
18
17 16 1 18 17 1 18 16 18 3 17 18 14 15 13 12 2 8 6 2 14 2 2 10 3 4 5 3 16 3 4 5 7 4 4 12 5 12 5 16 6 12 6 13 6 8 6 14 12 7 7 8 8 12 9 11 9 15 10 14 15 10 11 10 11 15
5
4 2 2 1 3 4 2 3 1 3 1 5 5 3
5
2 5 5 4 4 1 4 2 1 2 4 3 3 1
