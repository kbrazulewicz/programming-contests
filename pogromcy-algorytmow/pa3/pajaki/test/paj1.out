TAK
TAK
NIE
TAK
NIE
TAK
